packages/types/src/element/element.ts

```diff
@@ -65,4 +75,9 @@ export interface Element extends ElementBox {
   groupId?: string;
   border?: Border;
   borderRadius?: BorderRadius;
+  pageAttachment?: PageAttachment;
+  resource?: Resource;
+  overlay?: Pattern;
+  backgroundColor?: Pattern;
```

Comment:
This is only a property of shape and text elements, not all elements.

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12382/commits/a1cea5e2504a0731e448b3e5895b18eb6562016d